kalite/distributed/templates/distributed/base.html

```diff
@@ -140,6 +119,37 @@
             });
         </script>
 
+        <script>
+        window.sessionModel.set({
+          SEARCH_TOPICS_URL           : "{% url 'flat_topic_tree' lang_code=current_language %}",
+          STATUS_URL                  : "{% url 'status' %}",
+          FORCE_SYNC_URL              : "{% url 'api_force_sync' %}",
+
+          LOCAL_DEVICE_MANAGEMENT_URL : "{% url 'device_redirect' %}",
+          USER_LOGIN_URL              : "{% url 'login' %}",
+          SET_DEFAULT_LANGUAGE_URL    : "{% url 'set_default_language' %}",
+          DELETE_USERS_URL            : "{% url 'delete_users' %}",
+          DELETE_GROUPS_URL           : "{% url 'group_delete' %}",
+          MOVE_TO_GROUP_URL           : "{% url 'move_to_group' %}",
+          STATIC_URL                  : "{% static "" %}",
+
+          ALL_ASSESSMENT_ITEMS_URL    : "{% url 'api_dispatch_list' resource_name='assessment_item' %}",
+          GET_VIDEO_LOGS_URL          : "{% url 'api_dispatch_list' resource_name='videolog' %}",
+          GET_EXERCISE_LOGS_URL       : "{% url 'api_dispatch_list' resource_name='exerciselog' %}",
+          GET_CONTENT_LOGS_URL        : "{% url 'api_dispatch_list' resource_name='contentlog' %}",
+
+          KHAN_EXERCISES_SCRIPT_URL : "{% static 'perseus/ke/khan-exercise.js' %}",
```

Comment:
Haha, I like that even this one was kept unaligned from the others (like how it was before).